package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@zenginehq/zengine-sdk": "github:ZengineHQ/zengine-sdk",
+    "firebase": "^2.3.2",
```

Comment:
I noticed that Ted was using this, so I assume this works fine with our database secrets?

Reply:
Yep both 2.3.1 and 2.3.2 work fine, there's a bug in 2.3.1 that affected some things in the past.  Most plugins use 2.3.2 because of that.  If and when we figure out how to use the new sdk we can remove this....

Reply:
oh, I didn't realize this is the version we always used.